client/src/pages/Login/index.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import './login.css'
+import { Link } from 'react-router-dom'
+import { useNavigate } from 'react-router-dom'
+
+export default function Login() {
+  const [details, setDetails] = React.useState({email: '', password: '' })
+  const navigate = useNavigate()
+  const handleLogin= async (e) => {
+    e.preventDefault()
+    const {email, password } = details
+    const user = { email, password }
+    try {
+      const response = await fetch('http://localhost:4000/userLogin', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(user)
+      })
+      const json = await response.json()
+      if (json.token) {
+        alert('Login Successful')
+        localStorage.setItem('token', json.token)
+        navigate('/')
+      }
+      else{
+        alert('Invalid Credentials')
+      }
+    } catch (error) {
+      console.error('Error:', error)
+    }
+  }
+
+  return (
+    <>
+    
+    <div className='rectangleRight'></div>
+
+    <div className='rectanglePurpleRight'></div>
+
+    {/* Login Actual Div */}
+    <div className='loginDiv'>
+    <form className='formDiv'>
+      <h3>Doctor Verse</h3>
+      <div className='form-element'>
+        <label>E-mail</label>
+        <input name='email' className='input' type='text' autoComplete='off' onChange={e => setDetails({...details, email: e.target.value})} value={details.email} required style={{color:'white'}}></input>
+      </div>
+      <div className='form-element'>
+        <label>Password</label>
+        <input name='password' className='input' type='text' autoComplete='off' onChange={e => setDetails({...details, password: e.target.value})} value={details.password} required style={{color:'white'}}></input>
+      </div>
+      {/* Form Buttons */}
+      <div className='FormButtonsDiv'>
+        <div className='signUp'><button className='formButton' 
+        style={{
+          backgroundColor: 'white',
+          color: 'black',
+          border: '1px solid black',
+          borderRadius: '5px',
+          width: '100px',
```

Comment:
keeping height & width in pixels will keep the website responsive?

Reply:
changed in rem